### ISSUE TITLE
Updates with Abi

### ISSUE THREAD

**Issue by OlivierJM** [SparkEdUAB/SparkEd-Dev#26]
Follow up with @abnios on what is not properly working at the moment and how these can be fixed

**Comment by OlivierJM** [comment]
Thanks Abi, 

I have corrected the first one already
The second one for media files, I don't know exactly why but I can play the ones you have uploaded from here.

**Comment by OlivierJM** [comment]
The units are searchable I can confirm that, we can't change the password while offline because it needs to send an email.

**Comment by abnios** [comment]
Thank You, Oliver, Yes I tested the units, it is working on the deployed server version, I was testing on the prev pulled version. sorry about that.
for the first point, another user can view and delete courses. but can't edit the course and cannot see units under that course. I tested it now. I can view Math subject, but can not see units under it and can't edit maths, please let me give you my credential and test it. or you can also check by Biology on the dashboard. the good thing is the viewers on the home page can see all contents.

**Comment by OlivierJM** [comment]
I was just discussing with Craig about it. 
I will change as advised before the end of today.

**Comment by abnios** [comment]
Thank You Oliver.
What did you think about adding the Grade hierarchy before subject?? can we have that by today? and/or can you push all these changes to repo please???

**Comment by abnios** [comment]
Oliver what I think of is that,  just correct the bug issues and then to clone for the secondary school need from that one to a different branch, we might need this sparked version for programs that have one parent and many subject areas. please don't overright this version totally to replace to the secondary school one.

**Comment by OlivierJM** [comment]
I agree, so can we use the SparkEd-High for this we replace programs with grades, or I just create another branch and add a grade layer on top of subjects? I won't override the current setup

**Comment by abnios** [comment]
which will be easy.., updating all the changes to the SparkEd-High or add the Grade layer?, I think to test all the changes it would be nice to add the layer. what did you suggest?

**Comment by OlivierJM** [comment]
Yeah it could take time updating the SparkEd-High but I will eventually have to do that.

I can try and add the layer on top
